docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-slurm/slurm-known-issues.rst

```diff
@@ -170,6 +170,42 @@ Some constraints are due to differences in behavior between Docker and Singulari
    -  Enroot does not provide a mechanism for sharing containers. Each user must create any
       containers needed by their Determined experiments prior to creating the experiment.
 
+********************
+ Slurm Known Issues
+********************
+
+   -  A Determined experiment remains ``QUEUEUED`` for an extended period of time:
+
+      Inspect the details of your queued jobs using the Slurm ``scontrol show jobs`` command. If the
+      Slurm job is ``PENDING`` review the ``Reason`` code provided. See `JOB REASON CODES
+      <https://slurm.schedmd.com/squeue.html#SECTION_JOB-REASON-CODES>`__. Some common reasons are:
+
+      -  ``Resources``: Expected when resources are in use by other jobs. Otherwise, verify you have
+         not requested more resources (gpus, cpus, nodes, memory) than are available in your
+         cluster.
+
+      -  ``PartitionNodeLimit``: Ensure that the job is not requesting more nodes than ``MaxNodes``
+         of the partition.
+
+         Ensure that the ``MaxNodes`` setting for the partition is at least as high as the number of
+         GPUs in the partition. The ``MaxNodes`` value for a partition can be viewed in the
+         ``JOBS_SIZE`` column of the command:
+
+            .. code:: bash
+
+               sinfo -o "%15P %.5a %.10s %.8h %.6D %.11T %N %20G %E"
```

Comment:
Or just:

sinfo -O Partition,Size

Reply:
Yeah.    I included the others as they may be useful in general diagnosing this type of problem, but would be prettier with this syntax.   Updated to use -O instead.